projects/cups/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# Copyright 2019 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+./buildconf.sh
+./configure
+make clean
+make
```

Comment:
Does this build the fuzz targets and puts them into $OUT? 
Also, is there a seed corpus?

Reply:
Not yet.  I'm working on landing the fuzz target upstream.

Reply:
Removed leak

Reply:
Ok. Ping us when at least one target is landed. 
The rest looks good. Happy fuzzing!